src/components/ProfilePictures/ProfilePicture.vue

```diff
@@ -4,12 +4,16 @@
       v-if="user"
       :to="{name:'user', params: {userId: user.id}}"
     >
-      <RandomPicture
-        :name="user.displayName"
+      <RandomArt
+        :text="user.displayName"
         :seed="user.id"
         :size="size"
+        :width="size + 'px'"
+        :height="size + 'px'"
+        style="display: block; overflow: hidden"
+        :style="{ width: size + 'px' , height: size + 'px' }"
```

Comment:
I'm a bit confused here, why is it necessary to set width/height two times? `RandomArt` doesn't define the `width` and `height` props, are these native attributes?